src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/GetRecallNotificationComponentTest.kt

```diff
@@ -54,6 +55,39 @@ class GetRecallNotificationComponentTest : ComponentTestBase() {
     assertThat(response.content, equalTo(expectedBase64Pdf))
   }
 
+  @Test
+  fun `get recall notification returns merged recall summary and revocation order and then create recall notification creates new version but reuses existing revocation order`() {
```

Comment:
What about this test verifies "but reuses existing revocation order"?

Reply:
Well, there are 2 actions here. First one is the initial call to get recall notification, then we call the create endpoint which reuses. so it is wordy, but makes sense